### ISSUE TITLE
Integration Test Audit 1 - Feb 2020

### ISSUE THREAD

**Issue by jrbenny35** [mozilla/experimenter#2323]
I'll be going through the current automation plan for the integration tests, as well as the CI configuration and a few other things. Please add any thoughts here.

Works on #2221

**Comment by jrbenny35** [comment]
Docs are being added - #2317

**Comment by jrbenny35** [comment]
Changes:

- The tests can be run in parallel on CI
- Headless Firefox option should only be set for CI.
-

**Comment by jrbenny35** [comment]
@jaredlockhart Anything else here?

**Comment by jaredlockhart** [comment]
- Can we move the tests dir into the experimenter dir?
- Should this use fill_overview fixture?
  - https://github.com/mozilla/experimenter/blob/master/tests/integration/test_create_experiment.py#L8-L17
  - https://github.com/mozilla/experimenter/blob/master/tests/integration/test_create_experiment.py#L30-L37

**Comment by jrbenny35** [comment]
> All CSS locators should be constants? 
Some of these are chained off of other elements. Would you still want these to be added as a constant?

https://github.com/mozilla/experimenter/blob/master/app/tests/integration/pages/experiment_timeline_and_population.py#L71

**Comment by jaredlockhart** [comment]
Yeah I don't see why not?

**Comment by jaredlockhart** [comment]
Some pages define a page wait locator and don't use it:

https://github.com/mozilla/experimenter/blob/master/app/tests/integration/pages/experiment_detail.py#L13

https://github.com/mozilla/experimenter/blob/master/app/tests/integration/pages/experiment_objective_and_analysis.py#L18

https://github.com/mozilla/experimenter/blob/master/app/tests/integration/pages/home.py#L9

https://github.com/mozilla/experimenter/blob/master/app/tests/integration/pages/owned_experiments.py#L11

https://github.com/mozilla/experimenter/blob/master/app/tests/integration/pages/subscribed_experiments.py#L11

Some pages define a page wait locator and use the wrong locator:

https://github.com/mozilla/experimenter/blob/master/app/tests/integration/pages/experiment_design.py#L21-L27


Some pages don't define a page wait locator at all:

https://github.com/mozilla/experimenter/blob/master/app/tests/integration/pages/experiment_overview.py


Should be a class var:

https://github.com/mozilla/experimenter/blob/master/app/tests/integration/pages/experiment_timeline_and_population.py#L32

**Comment by jrbenny35** [comment]
> Should be a class var:
> 
> https://github.com/mozilla/experimenter/blob/master/app/tests/integration/pages/experiment_timeline_and_population.py#L32

I don't quite follow on this one.

**Comment by jaredlockhart** [comment]
Oh nevermind that I think I fixed it already in another PR

**Comment by jrbenny35** [comment]
With the closing of #2414 I think this first audit is done! If you agree @jaredlockhart please close this issue.

**Comment by jrbenny35** [comment]
Closed in #2454
